lib/tokenlist.cpp

```diff
@@ -485,6 +485,10 @@ static Token * skipDecl(Token *tok)
         } else if (Token::Match(vartok, "%var% [:=(]")) {
             return vartok;
         }
+        else if (Token::Match(vartok, "decltype ( auto ) ")) {
+            return vartok->next()->next()->next()->next();
```

Comment:
This could be written as `return vartok->tokAt(4);`

Reply:
fixed with `` 891440f ``